src/ansys/mapdl/core/mapdl_grpc.py

```diff
@@ -2265,8 +2260,8 @@ def _download_on_local(
 
     def _download_from_remote(
         self,
-        files: Union[str, List[str]],
-        target_dir: Optional[str] = None,
+        files: Union[str, List[str], Tuple[str, ...],
```

Comment:
I am sorry, the error comes from my suggestion
the closing square bracker is missing here:
```suggestion
        files: Union[str, List[str], Tuple[str, ...]],
```